lib/ui/playground/data/playground_tasks_datasource.dart

```diff
@@ -64,6 +68,53 @@ class PlayGroundTasksDataSource implements PlaygroundDataSource {
               child: ActionSheet(
                   viewModel: MockActionSheetViewModel.buildWithCheckBox(),
                   style: ActionSheetStyle.defaultStyle()))),
+      PlaygroundWidget(
+        title: 'TASK FARM-278 State carousel view',
+        child: Container(
+          height: 162,
+          child: CarouselView(
+            children: <Widget>[
```

Comment:
@LeeMatthewHiggins should I extract these children in another file and add it here? This file will be massive 😅

Reply:
yeah, maybe we can split by app area (tab)

Reply:
> Just some minor naming

Fixed!